soc/nxp/imxrt/imxrt118x/soc.c

```diff
@@ -269,6 +269,14 @@ static ALWAYS_INLINE void clock_init(void)
 
 #endif /* CONFIG_COUNTER_MCUX_GPT */
 
+#if !(DT_NODE_HAS_COMPAT(DT_CHOSEN(zephyr_flash), nxp_imx_flexspi)) &&              \
```

Comment:
usually the chosen flash is not the flexspi node, it's the flash node which is a child on the bus of the flexspi node.

Reply:
Yes, I correct to use flexspi_nor node.